src/PointwiseFunctions/AnalyticSolutions/RadiationTransport/M1Grey/ConstantM1.hpp

```diff
@@ -71,7 +71,7 @@ class ConstantM1 : public MarkAsAnalyticSolution {
   explicit ConstantM1(CkMigrateMessage* /*unused*/) noexcept {}
 
   // @{
-  /// Retrieve M1 variables at `(x, t)`
+  /// Retrieve variables templated on neutrino species at `(x, t)`
```

Comment:
Stressing and grouping by whether the tags have template parameters or not in the documentation seems odd.  It's not like it is possible to call the wrong one.

Reply:
It was mostly to match the division needed in the macros... but I can change this.